.travis.yml

```diff
@@ -19,17 +19,33 @@ jobs:
         - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - NPM_TEST=1
         - RELEASE=1
-    - php: 7.2
+    - php: 7.3
+      env:
+        - DB=MYSQL
+        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
+        - BEHAT_TEST=1
+        - REQUIRE_GRAPHQL="^3@dev"
+        - BEHAT_SUITE="admin"
+    - php: 7.3
+      env:
+        - DB=MYSQL
+        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
+        - BEHAT_TEST=1
+        - REQUIRE_GRAPHQL="^3@dev"
+        - BEHAT_SUITE="cms"
+    - php: 7.3
       env:
         - DB=MYSQL
         - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - BEHAT_TEST=1
+        - REQUIRE_GRAPHQL="^3@dev || 4.x-dev"
```

Comment:
I don't understand the rational for having a `3 || 4` constraint here. In practice, the 4 version will always be installed no matter what. Having a `3 || 4` constraint just adds useless ambiguity about what is being tested for each matrix entry.

Reply:
I'm not sure either at this point.  There would have been a rationale back in the day, presumably there was some cases where graphql 4 wasn't installable and we still wanted the job to still run so it would fallback to graphql 3

At this point though, yeah it doesn't really seem to make any sense

That said, there are several other modules that have `"^3@dev || 4.x-dev"` and I'd rather not have 2 different ways of saying 'run graphql 4'

Reply:
I've updated it to `^4@dev` - my understanding is that's pretty much the loosest constraint non-tag constraint there is so it should work with whatever recipes / modules define e.g. `^4.1 / 4.x-dev / 4.5.x-dev`